pkg/private/BUILD

```diff
@@ -57,7 +57,7 @@ py_library(
     ],
     srcs_version = "PY3",
     visibility = [
-        "//pkg:__pkg__",
+        "//:__subpackages__",
```

Comment:
Could this just be `//pkg:__subpackages__` instead?

I'm not opposed to this change as-is, though.

Reply:
I think the second one is too broad. Unless we want the internals visible to things like doc geneation.